src/components/timeline/_timeline.scss

```diff
@@ -0,0 +1,97 @@
+/* ==========================================================================
+  #TIMELINE
+  ========================================================================== */
+
+$timeline-badge-size-mobile: 16px;
+$timeline-badge-size-tablet: 20px;
+$timeline-badge-small-size-mobile: 12px;
+$timeline-badge-small-size-tablet: 16px;
+
+$timeline-border-width: 2px;
+
+.nhsapp-timeline {
+  @include nhsuk-responsive-margin(5, "bottom");
+  @include nhsuk-responsive-padding(2, "top");
+
+  list-style: none;
+  padding: 0;
+
+  &__item {
+    @include nhsuk-responsive-padding(5, "bottom");
+
+    display: flex;
+    margin-bottom: 0;
+    margin-left: 12px;
+    margin-top: -6px;
+    position: relative;
+
+    &:last-child {
+      padding: 0;
+
+      &:before {
+        border: none;
+      }
+    }
+
+    &:before {
+      border-left: $timeline-border-width solid $color_nhsuk-grey-3;
+      bottom: 0;
+      content: "";
+      display: block;
+      left: -$timeline-border-width;
+      position: absolute;
+      top: nhsuk-spacing(2);
+      width: $timeline-border-width;
+    }
+
+    &--past {
+      &:before {
+        border-color: $color_nhsuk-blue;
+      }
+    }
+  }
+
+  &__badge {
```

Comment:
Is the timeline icon a badge? We have [the badge component](https://nhsuk.github.io/nhsapp-frontend/components/badge/) already so might make sense to come up with a different name for this.

Reply:
As discussed offline, will need @davidhunter08 input